apps/spruce/src/pages/spawn/spawnHost/SpawnHostTableActions.test.tsx

```diff
@@ -203,6 +203,41 @@ describe("spawn host table", () => {
       screen.queryByDataCy("pause-unexpirable-host-button"),
     ).not.toBeInTheDocument();
   });
+
+  it("does not prompt user when permanent exemption is granted", () => {
+    const { Component } = RenderFakeToastContext(
+      <SpawnHostTable
+        hosts={[
+          {
+            ...baseSpawnHost,
+            sleepSchedule: {
+              ...defaultSleepSchedule,
+              nextStartTime: null,
+              timeZone: "America/New_York",
+              permanentlyExempt: true,
+            },
+          },
+        ]}
+      />,
+    );
+    render(
+      <MockedProvider
+        mocks={[
+          getSpruceConfigMock,
+          getUserSettingsMock,
+          instanceTypesMock,
+          myHostsMock,
+          myPublicKeysMock,
+          myVolumesQueryMock,
```

Comment:
nit. not really related to this PR but we could probably pull these out into a variable since it is repeated